src/queryResult/queryResultWebViewController.ts

```diff
@@ -93,7 +93,14 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
                     message.numberOfRows,
                 );
             let currentState = this.getQueryResultState(message.uri);
-            if (currentState.isExecutionPlan) {
+            if (
+                currentState.isExecutionPlan &&
```

Comment:
is this like our "prettier" wanting the code like this?  I don't know, it feels like LISP or something, ok, whatever. 😄

Reply:
Not LISP! but yeah the default formatter did that

Reply:
Yeahh... I want to bump the target printwidth for prettier so that it's a little less aggressive about wrapping lines.